core/src/main/scala/org/apache/gearpump/util/Util.scala

```diff
@@ -44,6 +45,11 @@ object Util {
     classpathList
   }
 
+  def getCurrentTime(): String = {
```

Comment:
Because we bind getCurrentTime with format "yyyy-MM-dd-hh-mm", I think it is too specific to put this in Util.